### DISCUSSION TITLE
Fattura Elettronica | show_paypal_button | API

### DISCUSSION THREAD

**Discussion by Mustino2022** [fattureincloud/api#343] [Q&A] [answered]
Buongiorno,
nella creazione della fattura elettronica tramite API impostando a true **show_paypal_button** e nel payments_list status not_paid come è possibile ricevere l'informazione quando la fattura viene pagata? Lo stato del pagamento cambia nella fattura a seguito del pagamento del cliente?

**Comment by valmoz** [comment, marked as answer]
Buongiorno @Mustino2022 
la pregherei la prossima volta di selezionare la categoria corretta delle discussion, che è "Q&A" per le domande come questa. 

In ogni caso, sì. Come spiegato in [questa guida](https://secure.fattureincloud.it/help/articolo/312-inserisci-pulsante-paga-paypal-documenti) il documento viene segnato automaticamente come saldato.

In futuro sarà possibile anche utilizzare i [Webhooks](https://developers.fattureincloud.it/docs/webhooks/) per essere aggiornati in tempo reale sullo stato dell'applicazione, ma al momento l'unico modo resta quello di recuperare il documento.

A disposizione
Mauro
